assets/js/webflow/import.js

```diff
@@ -72,6 +72,14 @@ ga('create', '{{ GOOGLE_ANALYTICS_ID }}', 'auto');\
 {% endif %}\
 ga('send', 'pageview');\
 ");
+  addScriptToBody(window, {
+    src: 'https://www.googletagmanager.com/gtag/js?id={{ GOOGLE_GA4_TAG }}',
```

Comment:
@michaelglenister is it right that we're adding Google Tag Manager but the variable is called `GOOGLE_GA4_TAG`? In other projects we use `TAG_MANAGER_CONTAINER_ID` for Google Tag Manager.

Reply:
My thought process was I wanted a prefix that placed the variables next to each other in dokku config. I think GOOGLE_TAG_MANAGER_ID is better unless we really want to standardize env variable names